sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusSessionsEndToEndTests.cs

```diff
@@ -588,24 +610,30 @@ public static async Task WaitForCancellation(CancellationToken cancellationToken
 
         public class ServiceBusMultipleTestJobsBase
         {
-            protected static bool firstReceived = false;
-            protected static bool secondReceived = false;
+            protected static volatile bool firstReceived = false;
```

Comment:
Did this help anything?

Reply:
not sure, but it seems like there is a potential race if not using volatile.